docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+User Guide
+=======
+SETA is a **desktop application for CS2103T Teaching Assistants** to track their students’ and tutorials’ details, and
+questions asked by students. SETA is optimized for use via a Command Line Interface (CLI) while still having the
+benefits of a Graphical User Interface (GUI). If you can type fast, SETA enables you to track your students, manage your
+tutorials and note down questions more effectively than traditional GUI apps.
+
+The purpose of this guide is to inform you what features are available in SETA, and how to use them to solve your 
+needs as a CS2103T TA.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: What is a Command Line Interface (CLI)?**<br>
+
+It is a programme that allows you to create, modify and delete files using a text-based user interface. In SETA, all 
+you have to do is to type in the command box and press enter to track your students and manage your questions and 
```

Comment:
Probably need to show what a command box is, probably give a photo here and like draw a red rectangle around the command box. Perhaps User interface can change to display instead?

Reply:
Ok, I'll rectify this